### DISCUSSION TITLE
No "tailwindCSS.experimental.classRegex" in Tailwind CSS IntelliSense anymore

### DISCUSSION THREAD

**Discussion by provokcx** [joe-bell/cva#315] [Q&A] [answered]
**Describe the bug**
Add the following to our settings.json doesn't work anymore:
```json
{
  "tailwindCSS.experimental.classRegex": [
    ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"],
    ["cx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
  ]
}
```

**Expected behavior**
Tailwind CSS IntelliSense should pickup the use of the `cva()` function.

Is there a workaround?

**Comment by oleksii-lukin** [comment]
I used regex from this issue
https://github.com/tailwindlabs/tailwindcss-intellisense/issues/719

```
    "tailwindCSS.experimental.classRegex": ["cva\\(([^)]*)\\)", ".*Styles", "[\"'`]([^\"'`]*).*?[\"'`]"],
```

and it worked for me

**Comment by joe-bell** [comment, marked as answer]
Should be fixed with https://github.com/joe-bell/cva/pull/332
